src/lib/FocusTrap.ts

```diff
@@ -0,0 +1,141 @@
+import { decodeKeydown } from './util';
+
+const FOCUSABLE_ELEMENTS = [
```

Comment:
How about `FOCUSABLE_SELECTORS`, since we usually use "element" to mean something else?